src/publish.js

```diff
@@ -14,6 +14,7 @@
 
 const yargsOpenwhisk = require('./yargs-openwhisk.js');
 const yargsFastly = require('./yargs-fastly.js');
+const yargsDebug = require('./yargs-debug.js');
```

Comment:
I don't think it is necessary to create an extra file for this option, as it can only be used during `hlx publish`.

Reply:
Yeah this is a good point! @kptdobe raised this point earlier; I will remove this